src/pages/Orgs/pages/OrgsNew/components/MetadataForm.tsx

```diff
@@ -73,67 +74,81 @@ export default function MetadataForm({ formProps, onOrgCreated, ...rest }: Props
-                label={FieldNames.Description}
-                errorMessage={getErrorMessage(FieldNames.Description)}
-                disabled={isFormDisabled}
-              />
-            </FormControl>
-          )}
-        />
-
-        <Controller
-          name={FieldNames.Domain}
-          control={control}
-          render={({ field }) => (
-            <FormControl>
-              <UiTextField
-                {...field}
-                label={FieldNames.Domain}
-                errorMessage={getErrorMessage(FieldNames.Domain)}
-                disabled={isFormDisabled}
-              />
-            </FormControl>
-          )}
-        />
-
-        <UiButton type='submit'>Next</UiButton>
+        <Stack spacing={6} bgcolor={palette.background.light} p={6} borderRadius={4} my={6}>
+          <Controller
+            name={FieldNames.Logo}
+            control={control}
+            render={({ field }) => (
+              <FormControl>
+                <UiImageUploader
+                  {...field}
+                  label='Upload logo'
+                  // errorMessage={getErrorMessage(FieldNames.Logo)}
+                  disabled={isFormDisabled}
+                />
+              </FormControl>
+            )}
+          />
+
+          <Controller
+            name={FieldNames.Domain}
+            control={control}
+            render={({ field }) => (
+              <FormControl>
+                <UiTextField
+                  {...field}
+                  label='Domain'
+                  placeholder='https:///'
+                  errorMessage={getErrorMessage(FieldNames.Domain)}
+                  disabled={isFormDisabled}
+                />
+              </FormControl>
+            )}
+          />
+
+          <Controller
+            name={FieldNames.Name}
+            control={control}
+            render={({ field }) => (
+              <FormControl>
+                <UiTextField
+                  {...field}
+                  label='Name'
+                  placeholder='Company name'
+                  errorMessage={getErrorMessage(FieldNames.Name)}
+                  disabled={isFormDisabled}
+                />
+              </FormControl>
+            )}
+          />
+
+          <Controller
+            name={FieldNames.Description}
+            control={control}
+            render={({ field }) => (
+              <FormControl>
+                <UiTextField
+                  {...field}
+                  label={'Description'}
+                  multiline
+                  rows={5}
+                  placeholder='Write a small description ?'
+                  errorMessage={getErrorMessage(FieldNames.Description)}
+                  disabled={isFormDisabled}
+                />
+              </FormControl>
+            )}
+          />
+        </Stack>
+
+        <UiButton
+          type='submit'
+          sx={{ '&.MuiButton-sizeMedium': { px: 8, py: 3.5, height: 'auto' } }}
```

Comment:
why changed here, not a default button?

Reply:
If I change the default button, it changes in other places. I don't need it